ucp/__init__.py

```diff
@@ -41,6 +41,14 @@
 logger = get_ucxpy_logger()
 
 
+try:
+    import uvloop
+
+    uvloop.install()
+except ImportError:
+    pass
```

Comment:
Per Jim's comment, perhaps this should be moved to the benchmarking script instead?